### PR TITLE
88 access broadband

### PR DESCRIPTION
**Pull request by td928**
## Overview

Use EDDT data shared by the Erica 

## load data
bulk of the work is actually taking place in the `load_clean_source_data` where the column names are renamed with a few dictionaries. It is important to note the sequence of the functions being run is relevant here because the race replacement must be the first. 

## access_broadband
this part of the function is simply pulling the right geography rows from the spreadsheet.

### CODE REVIEW

**Review by mbh329 — APPROVED** [review]
This all looks good, I tested it locally and the outputs and column names are good. I have one question - is the data reported at the household level? access_broadband_households suggests that it is but just curious because the numbers seem low but I also don't have a ref to how many households are in each borough/city/puma. This doesn't hold up anything as Erica calculated it for us

**Review by AmandaDoyle — APPROVED** [review]
Outputs are correct so I approve, but flagging the potential enhancements that Max pointed out

File: aggregate/quality_of_life/access_to_broadband.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        'sheet_name': "ACS15-19",
+        'usecols': "A, NM:QI",
+        "header": 0,
+        "nrows": 63
+    }
+
+    df = pd.read_excel(**read_excel_arg)
+
+    cols = [col.lower() for col in df.columns]
+    for code, race in race_suffix.items():
```
Comment (SashaWeinstein) [inline aggregate/quality_of_life/access_to_broadband.py:76]: Like this chunk of code, think it's readable

**Comment by SashaWeinstein** [comment]
This all looks good except that the column names aren't intuitive to me. I see from the issue that we are to return 
Total households
With a computer
With broadband internet

"Access_broadband_households" to me reads like households with broadband internet. If we've already finalized then maybe it's not worth changing but could cause issues later on

**Review by SashaWeinstein — APPROVED** [review]
Approved with same comments as Max, borough mappers and puma clean should come from `PUMA_helpers`

**Comment by SashaWeinstein** [comment]
lol wonderful Te pushed those changes the at the same time I made my comment

**Comment by td928** [comment]
Thanks! To @mbh329 @SashaWeinstein comment, I incorporated those new functions. The new output for puma looks good as well. I was hesitant about the column naming as well except they are specified quite explicitly in the issue #88. @AmandaDoyle is the thinking here that we will be able to explain it in the data dictionary?

**Comment by td928** [comment]
okay. New naming conventions updated. All output files were updated. @SashaWeinstein @AmandaDoyle @mbh329

**Review by SashaWeinstein — APPROVED** [review]
Great, these changes address all my concerns

**Pull request merged**